vxquery-xtest/results/xqts.txt

```diff
@@ -10366,7 +10366,7 @@ Expressions/SeqExpr/FilterExpr//K-FilterExpr-36, EXPECTED_RESULT_GOT_SAME_RESULT
 Expressions/SeqExpr/FilterExpr//K-FilterExpr-37, EXPECTED_RESULT_GOT_ERROR
 Expressions/SeqExpr/FilterExpr//K-FilterExpr-38, EXPECTED_ERROR_GOT_RESULT
 Expressions/SeqExpr/FilterExpr//K-FilterExpr-39, EXPECTED_RESULT_GOT_ERROR
-Expressions/SeqExpr/FilterExpr//K-FilterExpr-4, EXPECTED_ERROR_GOT_SAME_ERROR
+Expressions/SeqExpr/FilterExpr//K-FilterExpr-4, EXPECTED_ERROR_GOT_RESULT
```

Comment:
Interesting - we should add a note about this change of behavior to the description on the JSONiq extension for VXQuery. I think that the original spec makes JSONiq optional,  we we might not do this and so such a description would be very helpful.